src/models/callercalleemodel.cpp

```diff
@@ -42,30 +42,47 @@ QVariant CallerCalleeModel::headerCell(Columns column, int role)
             return Qt::DescendingOrder;
         }
     }
-    if (role != Qt::DisplayRole) {
+    if (role != Qt::DisplayRole && role != Qt::ToolTipRole) {
         return {};
     }
 
-    switch (column) {
-        case Symbol:
-            return tr("Symbol");
-        case Binary:
-            return tr("Binary");
-        case SelfCost:
-            return tr("Self Cost");
-        case InclusiveCost:
-            return tr("Inclusive Cost");
-        case Callers:
-            return tr("Callers");
-        case Callees:
-            return tr("Callees");
+    if (role == Qt::DisplayRole) {
+        switch (column) {
+            case Symbol:
+                return tr("Symbol");
+            case Binary:
+                return tr("Binary");
+            case SelfCost:
+                return tr("Self Cost");
+            case InclusiveCost:
+                return tr("Inclusive Cost");
+            case Callers:
+                return tr("Callers");
+            case Callees:
+                return tr("Callees");
+        }
+    } else if (role == Qt::ToolTipRole) {
+        switch (column) {
+            case Symbol:
+                return tr("The symbol's function name. May be empty when debug information is missing.");
+            case Binary:
+                return tr("The name of the executable the symbol resides in. May be empty when debug information is missing.");
+            case SelfCost:
+                return tr("The number of CPU cycles directly used by this symbol while inside the function.");
+            case InclusiveCost:
+                return tr("The number of CPU cycles used by this symbol both directly and indirectly. The inclusive cost includes the costs of all functions called by this symbol plus it's self cost.");
```

Comment:
The number of samples attributed to this symbol, both directly and indirectly. This includes the costs of all functions called by this symbol plus its self cost.